README.md

```diff
@@ -1,2 +1,73 @@
-# template-pkg
-A template repository for creating a Toit package
+# Package template
+
+A template repository for creating a Toit package.
+
+## Toit package
+Use `toit.pkg describe` or `toit pkg describe` (depending on which Toit
+variant you use) to see how https://pkg.toit.io extracts package
+information.
+
+Either add a `name: ...` entry to the package.yaml or change the title
+of this README to the package name.
+
+Either add a `description: ...` entry to the package.yaml or ensure
+that the first paragraph of this README can be used as a description.
+
+## Copyright
+Don't forget to update the copyright holder in the license files.
+There are (up to) three license files:
+- `LICENSE`: usually MIT
+- `examples/EXAMPLES_LICENSE`: usually BSD0
+- `tests/TESTS_LICENSE`: usually BSD0
+
+We recommend to use the following Copyright header in `src` files (with your
+copyright):
+
+```
+// Copyright (C) 2022 Toit language
+// Use of this source code is governed by an MIT-style license that can be
+// found in the package's LICENSE file.
+```
+
+Similarly, you can use the following header for tests and examples:
+```
+// Copyright (C) 2022 Toit language
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the tests/TESTS_LICENSE file.
+```
+and
+```
+// Copyright (C) 2022 Toit language
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the examples/EXAMPLES_LICENSE file.
+```
+
+## Local package
+Examples and tests can have different dependencies than the package. This is,
+why they have their own package.yaml/package.lock.
+
+Open the examples (resp. tests) folder with a separate instance of your IDE.
+For vscode you could just write `code examples`.
+
+Install this package as a local package.
+```
+cd examples
+toit.pkg init
```

Comment:
Maybe initialize the package as part of the template? I find it unlikely that you would ever have an examples folder in a package without needing the package.

Reply:
done.